src/test/java/uk/gov/hmcts/cmc/claimstore/services/ClaimServiceTest.java

```diff
@@ -49,9 +51,15 @@
     private static final String DEFENDANT_EMAIL = "defendant@email.com";
     private static final String INVALID_DEFENDANT_TOKEN = "You shall not pass!";
     private static final Claim claim = createClaimModel(VALID_APP, LETTER_HOLDER_ID);
-    private static final UserDetails validDefendant = new UserDetails(DEFENDANT_ID, DEFENDANT_EMAIL);
-    private static final UserDetails invalidDefendant = new UserDetails(-1L, DEFENDANT_EMAIL);
-    private static final UserDetails claimantDetails = new UserDetails(11L, SUBMITTER_EMAIL);
+
+    private static final UserDetails validDefendant
+        = new UserDetails(DEFENDANT_ID, DEFENDANT_EMAIL, SUBMITTER_FORENAME, SUBMITTER_SURNAME);
+
+    private static final UserDetails invalidDefendant
+        = new UserDetails(-1L, DEFENDANT_EMAIL, SUBMITTER_FORENAME, SUBMITTER_SURNAME);
```

Comment:
why does -1 matter to us? how is that invalid

Reply:
nod idea, existing code

Reply:
actioned

Reply:
The new builder can be used here.